Geometry/MTDNumberingBuilder/src/GeometricTimingDet.cc

```diff
@@ -223,5 +223,5 @@ GeometricTimingDet::Rotation GeometricTimingDet::rotationBounds() const {
 std::unique_ptr<Bounds> GeometricTimingDet::bounds() const {
   const std::vector<double>& par = params_;
   TrackerShapeToBounds shapeToBounds;
-  return std::unique_ptr<Bounds>(shapeToBounds.buildBounds(shape_, par));
+  return std::unique_ptr<Bounds>(shapeToBounds.buildBounds(cms::DDSolidShape(static_cast<int>(shape_)), par));
```

Comment:
@adewit @ianna there are two kind of volumes which do not seem to have a correspondent in DD4hep, do we have use of them and is this compatible?

Reply:
Hmm, thinking about it again I think you're right that this would break down if we had a ddpolycone_rrz/ddpolyhedra_rrz (in old dd) - the corresponding int would be 7 or 8 and that doesn't give a corresponding type in cms::DDSolidShape. Think there is a map to the single ddpolycone/ddpolyhedra that can be used to get this correct, need to check.

Reply:
@adewit Sorry, I had forgotten about the `DDSolidShape` map. It is in `​DetectorDescription/​DDCMS/​interface/​DDSolidShapes.h`. @ianna I think we need a new template function for a `cms::DDSolidShape` to old `DDSolidShape` mapping. Could you please advise?

Reply:
@cvuosalo and @ianna - I've added the inverse of the `value` function (called `value_reverse` to use with the `LegacySolidShapeMap` to do `cms::DDSolidShape`➔ `DDSolidShape`). That seems to do the job; is that what you had in mind?